packages/shared/src/components/cards/ad/AdList.tsx

```diff
@@ -38,47 +34,40 @@ const getLinkProps = ({
 };
 
 export const AdList = forwardRef(function AdCard(
-  { ad, onLinkClick, domProps, index, feedIndex }: AdCardProps,
+  { ad, onLinkClick, onRefresh, domProps, index, feedIndex }: AdCardProps,
   inViewRef: InViewRef,
 ): ReactElement {
   const { isEnrolledNotPlus } = usePlusSubscription();
-  const { ref } = useAutoRotatingAds(ad, index, feedIndex, inViewRef);
+  const { ref, refetch } = useAutoRotatingAds(ad, index, feedIndex, inViewRef);
 
   return (
     <FeedItemContainer
       domProps={domProps}
       ref={ref}
       data-testid="adItem"
       linkProps={getLinkProps({ ad, onLinkClick })}
-      flagProps={
-        isEnrolledNotPlus
-          ? undefined
-          : {
-              adAttribution: (
-                <AdAttribution ad={ad} className={{ typo: 'typo-caption1' }} />
-              ),
-              type: undefined,
-            }
-      }
     >
       <CardContent>
         <CardTitle
           className={classNames(
-            'mr-4 line-clamp-4 flex-1 font-bold typo-title3',
-            isEnrolledNotPlus && '!mt-0',
+            '!mt-0 mr-4 line-clamp-4 flex-1 font-bold typo-title3',
           )}
         >
           {ad.description}
+          <AdAttribution ad={ad} className={{ main: 'mb-2 mt-4 block' }} />
         </CardTitle>
-        <AdImage ad={ad} ImageComponent={CardImage} className="mt-4" />
+        <AdImage ad={ad} ImageComponent={CardImage} />
       </CardContent>
-      {isEnrolledNotPlus && (
-        <div className="z-1 flex items-center pt-4">
-          <AdAttribution ad={ad} />
-          <RemoveAd />
-        </div>
-      )}
-      <CardSpace />
+
+      <div className="z-1 flex items-center pt-2">
+        <AdRefresh
+          onClick={() => {
```

Comment:
Can't we do this inside, it seems the same in both instances?

Reply:
Not really. It's used for both feed and comment, and calls different functions